File: Python/a01/a01.py
# **----------------------------------------------------------------------------
# Author:      Kelvin Muturi
# Date:        October 6, 2024
# Assignment:  CS-161A Assignment a01
# Description: This program calculates the amount of Pizza required for a pizza
#              party and the total cost of the required pizza
# Inputs:      number of people as an integer
# Outputs:     number of Pizzas as an integer and their total cost as a double
#  -----------------------------------------------------------------------****/
import math

slicesPerPizza = 12
slicesPerPerson = 2
price = 14.95

print("Welcome to my pizza party Calculator!\n")

numPeople= int(input("Enter the number of people attending:"))

pizzas = math.ceil(numPeople * slicesPerPerson / slicesPerPizza)
cost = price * pizzas

print("You will need: ", pizzas)
print("The total cost is: $", cost)

print("Thank you!! Goodbye!")

File: CPP/a01/a01.cpp
/* This program calculates the amount of Pizza required for a pizza party
 * and the total cost of the required pizza
*/

#include <iostream>

using namespace std;

int main() {
    cout << "Welcome to my pizza party Calculator!" << endl;

    int numPeople;
    int pizzas;
    int slicesPerPizza = 12;
    int slicesPerPerson = 2;
    double price = 14.95;
    double cost;
    
    cout << "Enter the number of people attending: ";
    cin >> numPeople;

    pizzas = ((numPeople * slicesPerPerson) / slicesPerPizza);
    cost = price * pizzas;

    cout << "You will need: " << pizzas << " Pizzas." << endl;
    cout << "The total cost is: $" << cost << endl;

    return 0;
}